### PR TITLE
Fix: change exporting types

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)
- [x] It does not introduce a breaking change or has description for the breaking change

### Description

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -301,6 +301,44 @@ instance.setData(newData); // Call API of instance's public method
...
+    ],
+    data: [
+        {
+            name: 'Beautiful Lies',
+            artist: 'Birdy',
+            release: '2016.03.26',
+            genre: 'Pop'
+        }
+    ]
+});
```
Comment (dongwoo-kim) [inline README.md]: 이렇게 짧게 표시해 주는 게 좋을 것 같아요. 나머지 정보는 중요하지 않으니까요.

```js
import Grid = require('tui-grid');

var instance = new Grid({
  // ...options
});
```

File: README.md [hunk 2]
```diff
@@ -301,6 +301,44 @@ instance.setData(newData); // Call API of instance's public method
 Grid.applyTheme('striped'); // Call API of static method
 ```
 
+### TypeScript
+If you using TypeScript, You must `import module = require('module')` on importing.
+[`export =` and `import = require()`](https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require)
+
```
Comment (dongwoo-kim) [inline README.md]: 표현이 약간 어색해서 다듬어보았습니다.

If you are using TypeScript, you must use `import module = require('module')` to import the Grid module. See ["export = " and "import = require()"](https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require).

File: README.md (hunk 1 above)
Comment (jung-han) [reply README.md]: 수정하였습니다~

File: README.md (hunk 2 above)
Comment (jung-han) [reply README.md]: 수정하였습니다!

**Pull request merged**